src/helics/shared_api_library/ValueFederate.h

```diff
@@ -249,18 +248,33 @@ HELICS_EXPORT HelicsInput helicsFederateGetInput(HelicsFederate fed, const char*
 HELICS_EXPORT HelicsInput helicsFederateGetInputByIndex(HelicsFederate fed, int index, HelicsError* err);
 
 /**
- * Get an input object from a subscription target.
- *
- * @param fed The value federate object to use to get the publication.
- * @param key The name of the publication that a subscription is targeting.
- *
- * @param[in,out] err The error object to complete if there is an error.
+* Get an input object from a subscription target.
+* DEPRECATED: use helicsFederateInputByTarget instead
```

Comment:
Should be "helicsFederateGetInputByTarget", right?

Reply:
If my guess in my previous comment is right, there's a few other instances of this mistake in this file that need to be addressed.